diff --git a/jobs/consul_agent/templates/agent_ctl.sh.erb b/jobs/consul_agent/templates/agent_ctl.sh.erb
index 5c4463b..a7fe132 100644
--- a/jobs/consul_agent/templates/agent_ctl.sh.erb
+++ b/jobs/consul_agent/templates/agent_ctl.sh.erb
@@ -102,7 +102,7 @@ EOF
       # enabling updates and then updating it with resolvconf -u seems to add
       # dns entries from where the box was generated.
       if ! grep -q 127.0.0.1 /etc/resolv.conf; then
-        sed -i -e '1i nameserver 127.0.0.1' /etc/resolv.conf
+        echo -e "nameserver 127.0.0.1\n$(cat /etc/resolv.conf)" > /etc/resolv.conf
       fi
     fi
 
