diff --git a/jobs/consul_agent/templates/agent_ctl.sh.erb b/jobs/consul_agent/templates/agent_ctl.sh.erb
index f8d8cd4..e3279b0 100644
--- a/jobs/consul_agent/templates/agent_ctl.sh.erb
+++ b/jobs/consul_agent/templates/agent_ctl.sh.erb
@@ -47,7 +47,7 @@ function setup_resolvconf() {
 	  if [[ "$(stat -c "%s" "${resolvconf_file}")" = "0" ]]; then
 		  echo 'nameserver 127.0.0.1' > "${resolvconf_file}"
 	  else
-		  sed -i -e '1i nameserver 127.0.0.1' "${resolvconf_file}"
+		  echo -e "nameserver 127.0.0.1\n$(cat /etc/resolv.conf)" > /etc/resolv.conf
 	  fi
   fi
 
