diff --git a/images/scripts/run b/images/scripts/run
index 5d8dff5..f5c1a4d 100755
--- a/images/scripts/run
+++ b/images/scripts/run
@@ -49,7 +49,7 @@ chmod 1777 /tmp/cc_tmp
 cc_worker_ctl=/var/vcap/jobs/cloud_controller_ng/bin/cloud_controller_worker_ctl
 grep -q 'export TMPDIR=\/tmp\/cc_tmp' "$cc_worker_ctl" || sed -i '2iexport TMPDIR=/tmp/cc_tmp' "$cc_worker_ctl"
 
-start runsvdir || true
+/usr/sbin/runsvdir-start &
 
 while [[ $(total_services) = 0 ]]; do
   sleep 1
